dvc/fs/oss.py

```diff
@@ -68,25 +67,17 @@ def oss_service(self):
         logger.debug(f"key id: {self.key_id}")
         logger.debug(f"key secret: {self.key_secret}")
 
-        auth = oss2.Auth(self.key_id, self.key_secret)
-        bucket = oss2.Bucket(auth, self.endpoint, self.bucket)
-
-        # Ensure bucket exists
-        try:
-            bucket.get_bucket_info()
-        except oss2.exceptions.NoSuchBucket:
-            bucket.create_bucket(
-                oss2.BUCKET_ACL_PUBLIC_READ,
-                oss2.models.BucketCreateConfig(
-                    oss2.BUCKET_STORAGE_CLASS_STANDARD
-                ),
```

Comment:
These permissions are clearly bad :facepalm: Just shows how neglected oss has been :slightly_frowning_face:

Reply:
For the record, new and improved implementation is coming soon in https://github.com/iterative/ossfs thanks to @karajan1001 's work.